bundles/org.openhab.binding.nobohub/src/main/java/org/openhab/binding/nobohub/internal/NoboHubBindingConstants.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.nobohub.internal;
+
+import java.time.Duration;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link NoboHubBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Jørgen Austvik - Initial contribution
+ * @author Espen Fossen - Initial contribution
+ */
+@NonNullByDefault
+public class NoboHubBindingConstants {
+
+    private static final String BINDING_ID = "nobohub";
+
+    public static final String API_VERSION = "1.1";
+
+    public static final String NAME = "name";
```

Comment:
I would suggest to prefix property constants with "PROPERTY_".

Reply:
Fixed